docs/tasks/configure-pod-container/task-pv-claim.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
 spec:
   accessModes:
     - ReadWriteOnce
+  storageClassName: ""
```

Comment:
Not sure if I should use the beta annotation instead which is compatible with 1.5